code/datums/components/sign_language.dm

```diff
@@ -196,6 +197,11 @@
 
 	return SIGN_OKAY
 
+/// Spellcasting with sign language
+/datum/component/sign_language/proc/can_cast_spell()
+	if(check_signables_state() >= 1) // Cannot cast if anything but SIGN_OKAY
```

Comment:
```suggestion
	if(check_signables_state() >= SIGN_HANDS_FULL) // Cannot cast if anything but SIGN_OKAY
```

Reply:
I'm not _100%_ sure about making it possible to cast if you have one hand open. Part of the balance was making it more difficult to use by requiring your hands be free, but considering you can speak spells fine while gibbering, it does sort of fit I suppose.

Reply:
do vinylspider's review for this line instead, it works better